docs/spec/openapi.yml

```diff
@@ -2426,404 +2426,3049 @@ components:
     # please do not edit the definition below manually
     # -------------------------
     DatabaseCluster:
-      description: DatabaseCluster is the Schema for the databaseclusters API.
-      properties:
-        apiVersion:
-          description: |-
-            APIVersion defines the versioned schema of this representation of an object.
-            Servers should convert recognized schemas to the latest internal value, and
-            may reject unrecognized values.
-            More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources
-          type: string
-        kind:
-          description: |-
-            Kind is a string value representing the REST resource this object represents.
-            Servers may infer this from the endpoint the client submits requests to.
-            Cannot be updated.
-            In CamelCase.
-            More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds
-          type: string
-        metadata:
-          type: object
-        spec:
-          description: DatabaseClusterSpec defines the desired state of DatabaseCluster.
-          properties:
-            allowUnsafeConfiguration:
-              description: AllowUnsafeConfiguration field used to ensure that the
-                user can create configurations unfit for production use.
-              type: boolean
-            backup:
-              description: Backup is the backup specification
-              properties:
-                enabled:
-                  description: Enabled is a flag to enable backups
-                  type: boolean
-                pitr:
-                  description: PITR is the configuration of the point in time recovery
-                  properties:
-                    backupStorageName:
-                      description: BackupStorageName is the name of the BackupStorage
-                        where the PITR is enabled
-                      type: string
-                    enabled:
-                      description: Enabled is a flag to enable PITR
-                      type: boolean
-                    uploadIntervalSec:
-                      description: UploadIntervalSec number of seconds between the
-                        binlogs uploads
-                      type: integer
-                  required:
-                  - enabled
-                  type: object
-                schedules:
-                  description: Schedules is a list of backup schedules
-                  items:
-                    description: BackupSchedule is the backup schedule configuration.
+        description: DatabaseCluster is the Schema for the databaseclusters API.
+        properties:
```

Comment:
The schema is still valid, however the indent is bigger than it was before and as it is for other schemas, that's why the diff shows more changes than was actually done. Please update it

Reply:
updated